src/main/java/org/openstreetmap/atlas/geography/StringCompressedPolyLine.java

```diff
@@ -150,8 +187,9 @@ private String compress(final PolyLine points, final int precision0)
             // Encode the differences between the points
             encoded += encodeNumber(latitude - oldLatitude);
             final long deltaLongitude = longitude - oldLongitude;
-            if (deltaLongitude > MAXIMUM_DELTA_LONGITUDE)
+            if (Math.abs(deltaLongitude) > MAXIMUM_DELTA_LONGITUDE)
             {
+                logger.warn("Unable to use string compression for {}", points.toWkt());
```

Comment:
Normally it would be but this specific exception is always caught on line 77 of `StringCompressedPolyLine` (which triggers the fallback), so I thought it would be worth logging. I believe catching the exception explicitly hides it from log output (since it is a checked exception and we are not dumping the stack trace in the `catch` block).